src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -90,8 +89,7 @@ static RuntimeMetrics()
         // Either Environment.Version is not 6 or (it's 6 but internal API GC.GetGenerationSize is valid)
         if (!isCodeRunningOnBuggyRuntimeVersion || getGenerationSize != null)
         {
-            // TODO: change to ObservableUpDownCounter
-            MeterInstance.CreateObservableGauge(
+            MeterInstance.CreateObservableUpDownCounter(
```

Comment:
Do you think this is a breaking change? For sure OTLP Exporters see dif. data type now, but exporter like Prometheus would be fine... 
@alanwest Can you check from NR stand point, if this will be breaking for anyone sending to NR via OTLP?

Reply:
Apologies for delay. No this is not breaking for New Relic. NR has its own gauge data type. Both OTLP gauges and OTLP non-monotonic sums are transformed to a NR gauge, so there will be no difference for end users querying this data in NR.

Though, it may be possible this could be breaking for other backends.

Reply:
We can bring this to the next SIG to make a call about do we need to major version bump or not.

Reply:
@lachmatt checked changes with Splunk OTLP endpoint and it is working correctly.

Reply:
BTW I would consider it as a bugfix not a breaking change. There were no possibility to set it correctly with older dependencies.